content/en/security/code_security/static_analysis/ai_enhanced_sast.md

```diff
@@ -0,0 +1,196 @@
+### Campain in progress
+
+When you click **Create Campaign**, Bits Dev Agent does the following:
+
+1. Bits Dev Agent loads SAST findings for the selected repo(s), path(s), and rule.
+2. Uses the [Bits Dev Agent][10] to generate patches for each group of findings.
+3. Creates PRs according to your session rules.
+4. If Bits Dev notifications are turned on, PRs will automatically be sent via Slack to relevant stakeholders.
+5. Lets you review, edit, and merge fixes using GitHub.
+
+The campain page shows whether the AI is actively remediating real findings, and how many have been remediated or pending.
+
+You can click a session to view the code changes in more detail and chat with the [Bits Dev Agent][10] to ask for changes.
+
+### Session details
+
+A remediation session shows the full lifecycle of an AI-generated fix. It includes the original security finding, a proposed code change, an explanation of how and why the AI made the fix, and if enabled, CI results from applying the patch. 
+
+Session details make each remediation transparent, reviewable, and auditable, helping you safely adopt AI in your secure development workflow.
+
+Session details include the following:
+
+- Header: Identifies the campaign, time of session creation, and affected branch, file, or PR.
+- Title: Summarizes the remediation goal based on the vulnerability being fixed.
+- **Dev Agent Campaign, Model name, PR ID**: Indicates the AI model used, campaign source, and related PR metadata.
+- Right panel:
+  - **Suggested code change**: Displays a side-by-side comparison of the vulnerable code and the AI-generated patch.
+  - **View Pull Request**: Opens the linked GitHub PR to review or merge the proposed changes.
+- Left panel displays the chat message history, for example:
+  - **Fix the following static analysis violation**: Explains the triggered rule, the security risk, and why the original code is unsafe.
+  - **Fixing process recap**: Shows exactly how the AI read the code, understood the context, chose its approach, and applied the fix. This is helpful for auditability, compliance, and trust. You can confirm that the AI isn't rewriting code blindly, but applying defensible and explainable patterns.
+  - **CI via GitHub**: Describes whether the AI-generated patch breaks anything downstream, and includes full error logs. This helps you validate that a fix is not only secure but also safe to deploy, without needing to leave the platform.
+  - **Summary**: Recaps the impact of the fix and provides next steps or guidance if tests failed or PR needs to be rebased.
+  - Bits chat field: Lets you interactively refine the fix or ask the AI follow-up questions. This makes remediation collaborative and tunable, giving security engineers control without needing to write the patch themselves.
+
+
+#### Scope
+
+AI-powered remediation is limited to SAST only for vulnerabilities within the OWASP Top 10 across all languages. It only applies to findings on the default branch.
```

Comment:
Might be worthwhile to turn this into an alert box near the beginning of the "AI-powered remediation" section, so readers don't have to read/scroll through the whole section only to find out it might not cover their use case.